app/controllers/StudyController.java

```diff
@@ -38,7 +33,8 @@ public void setUserProfileService(UserProfileService userProfileService) {
     public Result getStudyForResearcher() throws Exception {
         UserSession session = getAuthenticatedResearcherSession();
         Study study = studyService.getStudy(session.getStudyIdentifier());
-        return okResult(new StudyInfo(study));
+        
+        return ok(DynamoStudy.STUDY_WRITER.writeValueAsString(study));
```

Comment:
This tightly couples the StudyController with the DynamoStudy. Can we move STUDY_WRITER to the base Study class? Or alternatively make it a method that's overriden in DynamoStudy.